content/fundamental-redirects.js

```diff
@@ -0,0 +1,1145 @@
+    ({ sample_path }) =>
+      `https://media.prod.mdn.mozit.cloud/samples/${sample_path}`,
+    { permanent: false }
+  ),
+  // Bug 887428 - Misprinted URL in promo materials
+  // RewriteRule ^Firefox_OS/Security$ docs/Mozilla/Firefox_OS/Security
+  // [R=301,L,NC]
+  redirect(/^Firefox_OS\/Security$/i, "/docs/Mozilla/Firefox_OS/Security", {
+    permanent: true,
+  }),
+  // Old landing pages. The regex, adapted from Bedrock, captures locale prefixes.
+  // RewriteRule ^(\w{2,3}(?:-\w{2})?/)?mobile/?$ /$1docs/Mozilla/Mobile
+  // [R=301,L]
+  localeRedirect(/^mobile\/?$/i, "/docs/Mozilla/Mobile", { permanent: true }),
+  // RewriteRule ^(\w{2,3}(?:-\w{2})?/)?addons/?$ /$1Add-ons [R=301,L]
+  localeRedirect(/^addons\/?$/i, "/Add-ons", { permanent: true }),
+  // RewriteRule ^(\w{2,3}(?:-\w{2})?/)?mozilla/?$ /$1docs/Mozilla [R=301,L]
+  localeRedirect(/^mozilla\/?$/i, "/docs/Mozilla", { permanent: true }),
+  // RewriteRule ^(\w{2,3}(?:-\w{2})?/)?web/?$ /$1docs/Web [R=301,L]
+  localeRedirect(/^web\/?$/i, "/docs/Web", { permanent: true }),
+  // RewriteRule ^(\w{2,3}(?:-\w{2})?/)?learn/html5/?$
+  // /$1docs/Web/Guide/HTML/HTML5 [R=301,L]
+  localeRedirect(/^learn\/html5\/?$/i, "/docs/Web/Guide/HTML/HTML5", {
+    permanent: true,
+  }),
+  // Some blanket section moves / renames
+  // RewriteRule ^En/JavaScript/Reference/Objects/Array$
+  // en-US/docs/JavaScript/Reference/Global_Objects/Array [R=301,L,NC]
+  redirect(
+    /^En\/JavaScript\/Reference\/Objects\/Array$/i,
+    "/en-US/docs/JavaScript/Reference/Global_Objects/Array",
+    { permanent: true }
+  ),
+  // RewriteRule ^En/JavaScript/Reference/Objects$
+  // en-US/docs/JavaScript/Reference/Global_Objects/Object [R=301,L,NC]
+  redirect(
+    /^En\/JavaScript\/Reference\/Objects$/i,
+    "/en-US/docs/JavaScript/Reference/Global_Objects/Object",
+    { permanent: true }
+  ),
+  // RewriteRule ^En/Core_JavaScript_1\.5_Reference/Objects/(.*)
+  // en-US/docs/JavaScript/Reference/Global_Objects/$1 [R=301,L,NC]
+  redirect(
+    /^En\/Core_JavaScript_1\.5_Reference\/Objects\/(?<suffix>.*)$/i,
+    ({ suffix }) => `/en-US/docs/JavaScript/Reference/Global_Objects/${suffix}`,
+    { permanent: true }
+  ),
+  // RewriteRule ^En/Core_JavaScript_1\.5_Reference/(.*)
+  // en-US/docs/JavaScript/Reference/$1 [R=301,L,NC]
+  redirect(
+    /^En\/Core_JavaScript_1\.5_Reference\/(?<suffix>.*)$/i,
+    ({ suffix }) => `/en-US/docs/JavaScript/Reference/${suffix}`,
+    { permanent: true }
+  ),
+  // RewriteRule ^([\w\-]*)/HTML5$ $1/docs/HTML/HTML5 [R=301,L,NC]
+  localeRedirect(/^HTML5$/i, "/docs/HTML/HTML5", { permanent: true }),
+  // RewriteRule web-tech/2008/09/12/css-transforms
+  // /docs/CSS/Using_CSS_transforms [R=301,L]
+  redirect(
+    /^web-tech\/2008\/09\/12\/css-transforms$/i,
+    "/docs/CSS/Using_CSS_transforms",
+    { permanent: true }
+  ),
+  // RewriteRule ^([\w\-]*)/docs/?$ $1/docs/Web [R=301,L,NC]
+  localeRedirect(/^\/docs\/?$/i, "/docs/Web", { permanent: true }),
```

Comment:
I hacked the [kuma headless redirect tests](https://github.com/mdn/kuma/blob/master/tests/headless/test_redirects.py) to run against the local preview server (`pytest --base-url http://localhost:5000 tests/headless/test_redirects.py`), and discovered this one.

```suggestion
  localeRedirect(/^docs\/?$/i, "/docs/Web", { permanent: true }),
```